### ISSUE TITLE
So strange the loss value

### ISSUE THREAD

**Issue by forwardwfg** [qqwweee/keras-yolo3#329]
When I train COCO dataset, the loss value suddenly becomes about -150000  from about 50, and goes larger and larger toward negative direction. I am very confused about it. Can you give me some advice? Thanks!

**Comment by YunYang1994** [comment]
This repo does not support to train your dataset at all .

**Comment by forwardwfg** [comment]
@YunYang1994 could you tell me why not support? please.
